samples/highcharts/blog/nw-landing-page/demo.html

```diff
@@ -0,0 +1,44 @@
+
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/modules/networkgraph.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/stock/modules/data.js"></script>
+<script src="https://code.highcharts.com/modules/item-series.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"></script>
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet">
+
+</head>
+<body>
+
+ <div id="content" class="site-content">
+	 
+		<div class="row" ><!-- style="background-color: #47475c" -->
+	 		<div class="col-lg-12 d-lg-flex justify-content-center bkgWhite" >
```

Comment:
It seems like not all sizes work perfectly - on a wide screen I see 6 charts side by side. When reducing the width of my window gradually, the sixth and the fifth chart are partially clipped before they disappear. Then when below 768px, only four charts are visible, but I assume that is intentional, to avoid taking up too much vertical space.

Reply:
Yes, I intentionally clipped the container on small screens to limit it to 4 graphs.

Reply:
Okay, so then we have a problem between 768 and the full container width.

Reply:
I fixed the CSS so all charts display regardless of the screen size.